packages/react/src/index.ts

```diff
@@ -16,6 +16,7 @@ export interface ChatHook {
   messagesContainerRef: Ref<HTMLDivElement>;
   waiting: boolean;
   setInputValue: (val: string) => void;
+  scrollToBottom: () => void;
```

Comment:
Perform all changes in this file in the `packages/preact/src/index.ts` file as well. There is a comment marking at what point in the file you can just do a full-on copy-paste: https://github.com/nlxai/chat-sdk/blob/master/packages/react/src/index.ts#L3

Reply:
done